components/RichText/RichText.module.scss

```diff
@@ -54,12 +54,20 @@
     color: var(--color--indigo);
     font-weight: $weight--extrabold;
     position: relative;
-    text-decoration: none;
-    border-bottom: 2px solid var(--color--underline);
-    text-underline-position: under;
+    text-decoration: underline;
+    text-decoration-thickness: 2px;
+    text-underline-offset: 2px;
+    text-decoration-color: var(--color--underline);
 
     &:hover {
-        border-bottom: 2px solid var(--color--lagoon);
+        text-decoration-color: var(--color--lagoon);
+    }
+
+    @include hcm() {
+        &:hover,
+        &:focus {
+            @include hcm-underline-hover();
+        }
```

Comment:
Suggestion: Please could you also add this underline style rule to `Job.module.scss` line 44

Reply:
Yes I have done now - I've made the job rich text completely consistent with this - i.e. using underline rather than a bottom border for links in 'normal' mode as well as in high contrast mode.